tests/python/frontend/pytorch/qnn_test.py

```diff
@@ -29,6 +29,9 @@
 import tvm
 import tvm.testing
 from tvm import relay
+from tvm.contrib import graph_runtime
```

Comment:
This is a test script for pytorch frontend... Please move these changes to `relay/test_op_qnn_dequantize.py`

Reply:
Moved - thanks for pointing that out